colorama/ansitowin32.py

```diff
@@ -13,6 +13,44 @@
     winterm = WinTerm()
 
 
+def is_msys_cygwin_tty(stream):
+    # https://github.com/msys2/MINGW-packages/pull/2675/files
```

Comment:
Not a biggie, but I'd vote for putting this URL in the commit message (with a sentence explaining what it is, as you have done) not in the code. If people want more info about the function and where it came from, they can git blame it, and will find the URL.

Reply:
I've changed the commit message